src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/ApplicationsControllerTests.cs

```diff
@@ -153,5 +153,45 @@ public void POST_Application_SelectedActionIsDecline_ThrowsNotImplementedExcepti
             // Act/Assert
             Assert.ThrowsAsync<System.NotImplementedException>(() => _controller.Application(request));
         }
+
+        [Test]
+        public async Task GET_Withdrawn_ApplicationExists_ReturnsViewAndModel()
+        {
+            // Arrange
+            var request = _fixture.Create<WithdrawnRequest>();
+            _orchestrator
+                .Setup(x => x.GetWithdrawnViewModel(It.Is<WithdrawnRequest>(y => y == request)))
+                .ReturnsAsync(new WithdrawnViewModel());
+
+            // Act
+            var actionResult = await _controller.Withdrawn(request);
```

Comment:
var viewResult = await _controller.Withdrawn(request) as ViewResult;
var withdrawnViewModel = viewResult?.Model as WithdrawnViewModel;

Reply:
I know I _could_ do this, but it was intentional - it was written this way to easily figure out why the test was failing.

For example:

* If I changed the result type, the first `Assert.NotNull` would fail;
 * If I changed the view model type, the test would fail on the last `Assert.NotNull`

I know it seems verbose, but it's just meant to be clear and self-documenting. I'd agree that if this were non-test code, you wouldn't want to write it like this!